supported-libraries-schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "This will be populated with actual link once uploaded",
+  "title": "Schema for the supported libraries list JSON",
+  "type": "array",
+  "default": [],
+  "items": {
+    "type": "object",
+    "required": [
+      "artifact",
+      "minimal_version",
+      "metadata_location",
+      "support_level"
+    ],
+    "properties": {
+      "artifact": {
+        "type": "string",
+        "default": "",
+        "title": "The name of the metadata"
+      },
+      "minimal_version": {
+        "type": "string",
+        "default": "",
+        "title": "Minimal supported version"
+      },
+      "maximal_version": {
+        "type": "string",
+        "default": "",
+        "title": "Maximal supported version",
+        "description": "If not defined, it is assumed to be supported for all versions starting from the minimum_version"
+      },
+      "metadata_location": {
+        "type": "string",
+        "default": "",
+        "title": "Web location of provided metadata"
+      },
+      "support_level": {
+        "type": "string",
+        "enum": [
+          "Unsupported",
+          "Community Supported",
+          "Supported"
+        ],
+        "default": "Unsupported",
+        "title": "Metadata support level"
+      }
+    }
+  },
+  "examples": [
```

Comment:
I don't think the example should include metadata from this repo because metadata from this repo should not be mentioned in this file at all (otherwise, we'll have to keep it in sync with the actual metadata under `metadata/`).

Can we use something representative like [netty](https://github.com/netty/netty/tree/main/common/src/main/resources/META-INF/native-image)?

Reply:
We will update examples once we have clear idea of how to use all properties